### PR TITLE
UL27: flagged rule change

### PR DESCRIPTION
**Pull request by bemailloux**
While writing up the definition for "isFlagged" for Flagged Quiz Questions, John realized the current state didn't make sense from a domain perspective.

The new flagged definition is the following:
```
if (upper27Value < upper27Threshold) then flag upper27Value (unchanged rule)
-> this indicates that a question was too hard

if (lower27Value > lower27Threshold) then flag lower27Value (changed rule)
-> this indicates that a question was too easy
```

Quality notes
* Testing
  * [x] Verify that U27 column is flagged if value is less than threshold
  * [x] Verify that L27 column is flagged if value is greater than threshold
  * [x] Verify label and description text
* Security, UX, docs, perf, devops: N/A

FYI @Brightspace/draco-vis

### CODE REVIEW

File: components/settings/quiz-items-settings-list.js [hunk 1]
```diff
@@ -352,6 +365,7 @@ class QuizItemsSettingsList extends RtlMixin(Localizer(LitElement)) {
 
 	get settings() {
 		return Object.keys(allQuizItemsMetrics).reduce((settings, key) => {
+			// TODO DeMorgan's law this
```
Comment (bemailloux) [inline components/settings/quiz-items-settings-list.js:368]: TODO

File: locales/en.js [hunk 2]
```diff
@@ -116,6 +116,7 @@ export default {
 	"settings:averageScoreColumnDescription": "The Average Score metric shows the average across all learners' responses. The metric is calculated as a percentage using the score for each user's first attempt at the quiz. The data point will display in the Quiz Detail Table.",
 	"settings:flagDataPoint": "Flag if data point is lower than ",
 	"settings:flagDataPointHigh": "Flag if data point is higher than ",
+	"settings:flagDataPointHighOrLow": "TODO",
```
Comment (bemailloux) [inline locales/en.js:119]: TODO (waiting for Kevin)

**Review by Vitalii-Misechko — APPROVED** [review]
LGTM!

File: components/settings/quiz-items-settings-list.js [hunk 3]
```diff
@@ -157,58 +156,76 @@ class QuizItemsSettingsList extends RtlMixin(Localizer(LitElement)) {
...
-							@invalid-change="${this._invalidChange}"
-						></d2l-input-number>
-					` : nothing}
+					${this._renderInputField(colmName, input)}
 				</div>
 			</d2l-insights-settings-list-item>
 		`;
 	}
 
+	_renderInputField(colmName, input) {
```
Comment (NicholasHallman) [inline components/settings/quiz-items-settings-list.js:183]: Nice!

**Review by NicholasHallman — APPROVED** [review]

File: components/settings/quiz-items-settings-list.js (hunk 3 above)
Comment (bemailloux) [reply components/settings/quiz-items-settings-list.js:183]: Thanks. I feel like this ought to be part of `d2l-insights-settings-list-item` but, that refactor ended up being a fair bit larger than I wanted to commit to atm

File: test/visual-diff/settings-view.visual-diff.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ const VisualDiff = require('@brightspace-ui/visual-diff');
 		it('Mobile', async function() {
 			await page.setViewport({
 				width: 682,
-				height: 3000,
+				height: 4320,
```
Comment (bemailloux) [inline test/visual-diff/settings-view.visual-diff.js:45]: This change wasn't intended - but I didn't feel like bombarding everyone with more visual diff notifications by changing it :)

**Review by bbabu01 — APPROVED** [review]
LGTM

File: components/settings/quiz-items-settings-list.js (hunk 3 above)
Comment (NicholasHallman) [reply components/settings/quiz-items-settings-list.js:183]: I agree, changing that component from a basic template to a full fledged component would be really useful.

**Pull request merged**